cmd/postgres/main.go

```diff
@@ -39,7 +39,8 @@ func buildDatabase(repo repository.Repository) *derivative.PostgresClient {
 		WithPassword(os.Getenv("RDS_PASSWORD")).
 		WithDbname(os.Getenv("RDS_DB_NAME")).
 		WithHost(os.Getenv("RDS_HOSTNAME")).
-		WithPort(os.Getenv("RDS_PORT"))
+		WithPort(os.Getenv("RDS_PORT")).
+		WithSSL(os.Getenv("RDS_SSL") == "" || strings.ToLower(os.Getenv("RDS_SSL")) == "true")
 
```

Comment:
If you're setting `RDS_SSL` to false whenever you use it (i.e. above in the Makefile), why do you need to test for it like this? I feel like I must be missing something here...

Reply:
In production, SSL is true and RDS_SSL is not provided (hence ""). It only needs to be set to false in development.